### PR TITLE
Fix some broken TranslateSpec tests

### PR DESCRIPTION
**Pull request by Mingun**
It seems that the tests did not even run when making changes...

### CODE REVIEW

**Comment by Mingun** [comment]
@GreyCat , @generalmimon this one seems easy to check -- I've found almost all commits that introduced bugs

File: jvm/src/test/scala/io/kaitai/struct/translators/TranslatorSpec.scala [hunk 1]
```diff
@@ -596,6 +599,14 @@ class TranslatorSpec extends FunSuite {
   // sizeof of fixed user type
   everybody("bitsizeof<block>", "56", CalcIntType)
 
+  /**
+    * Checks translation of expression `src` into target languages
+    *
+    * @param src Kaitai struct expression to translate
```
Comment (generalmimon) [inline jvm/src/test/scala/io/kaitai/struct/translators/TranslatorSpec.scala:605]:
```suggestion
    * @param src KS expression to translate
```

File: jvm/src/test/scala/io/kaitai/struct/translators/TranslatorSpec.scala [hunk 2]
```diff
@@ -633,7 +643,7 @@ class TranslatorSpec extends FunSuite {
                 }
                 actResult2 should be(expResult)
               case None =>
-                fail("no expected result")
+                fail(s"no expected result, but result is ${tr.translate(e)}")
```
Comment (generalmimon) [inline jvm/src/test/scala/io/kaitai/struct/translators/TranslatorSpec.scala:646]:
```suggestion
                fail(s"no expected result, but actual result is ${tr.translate(e)}")
```

File: jvm/src/test/scala/io/kaitai/struct/translators/TranslatorSpec.scala [hunk 3]
```diff
@@ -596,6 +599,14 @@ class TranslatorSpec extends FunSuite {
   // sizeof of fixed user type
   everybody("bitsizeof<block>", "56", CalcIntType)
 
+  /**
+    * Checks translation of expression `src` into target languages
+    *
+    * @param src Kaitai struct expression to translate
+    * @param tp Type model that provides information about used user-defined types in expression
+    * @param expType Expected type that should be detected by translator
+    * @param expOut Map with expected outputs of each language
```
Comment (generalmimon) [inline jvm/src/test/scala/io/kaitai/struct/translators/TranslatorSpec.scala:608]:
```suggestion
    * @param expOut Map with expected outputs for each language
```

File: jvm/src/test/scala/io/kaitai/struct/translators/TranslatorSpec.scala [hunk 4]
```diff
@@ -596,6 +599,14 @@ class TranslatorSpec extends FunSuite {
   // sizeof of fixed user type
   everybody("bitsizeof<block>", "56", CalcIntType)
 
+  /**
+    * Checks translation of expression `src` into target languages
+    *
+    * @param src Kaitai struct expression to translate
+    * @param tp Type model that provides information about used user-defined types in expression
+    * @param expType Expected type that should be detected by translator
```
Comment (generalmimon) [inline jvm/src/test/scala/io/kaitai/struct/translators/TranslatorSpec.scala:607]:
```suggestion
    * @param expType Expected type that should be detected by [[TypeDetector]]
```

File: jvm/src/test/scala/io/kaitai/struct/translators/TranslatorSpec.scala [hunk 5]
```diff
@@ -622,7 +633,6 @@ class TranslatorSpec extends FunSuite {
       test(s"$langName:$src", Tag(langName), Tag(src)) {
         eo match {
           case Some(e) =>
-            val tr: AbstractTranslator with TypeDetector = langs(langObj)
```
Comment (generalmimon) [inline jvm/src/test/scala/io/kaitai/struct/translators/TranslatorSpec.scala:625]: It seems probably OK, but can you explain why did you delete this line?

**Review by generalmimon — CHANGES_REQUESTED** [review]

File: jvm/src/test/scala/io/kaitai/struct/translators/TranslatorSpec.scala (hunk 5 above)
Comment (Mingun) [reply jvm/src/test/scala/io/kaitai/struct/translators/TranslatorSpec.scala:625]: Just because `tr` already in scope from line 620

**Review by generalmimon — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**